### PR TITLE
Update hook component documentation

### PR DESCRIPTION
**Pull request by eternoendless**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 8.x
| Description?  | This change updates and expands documentation about the Hook component
| Fixed ticket? | N/A

### CODE REVIEW

File: development/components/hook/dispatching-hook.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Dispatching a Hook
+weight: 20
+---
+
+# Dispatching a Hook
+
+Hooks are placed everywhere throughout the software. Whenever a given hook is dispatched, the Hook component will check which modules subscribed to that hook and notify them by successively calling each subscriber's callback.
+
+Hooks are dispatched using the `exec()` static function.
```
Comment (matks) [inline development/components/hook/dispatching-hook.md]: Not to be confused with https://www.php.net/manual/en/function.exec.php 😆

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
@eternoendless there is a gift conflict. But the PR content is good.

File: development/components/hook/dispatching-hook.md (hunk 1 above)
Comment (eternoendless) [reply development/components/hook/dispatching-hook.md]: Fixed

**Comment by eternoendless** [comment]
Rebased and fixed

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @eternoendless

**Pull request merged**